third_party/terraform/utils/error_retry_predicates.go

```diff
@@ -1,23 +1,82 @@
 package google
 
 import (
-	"google.golang.org/api/googleapi"
+	"fmt"
 	"log"
+	"net/url"
 	"strings"
+
+	"google.golang.org/api/googleapi"
 )
 
+type RetryErrorPredicateFunc func(error) (bool, string)
+
+var defaultErrorRetryPredicates = []RetryErrorPredicateFunc{
```

Comment:
nit: It could be helpful to block-comment above / below the defaults to try to encourage future defaults to end up in that block.